resources/public/api/conf/2.0/application.raml

```diff
@@ -102,7 +102,7 @@ traits:
 
         This request is also called an ‘Declaration Data’ E_DEC_DAT (IE015) message.
 
-        A successful response will contain a URI and a departure ID which you can use to retrieve messages related to the movement.
+        A successful response will contain a URI which you can use to retrieve the message status. A successful response can also include an optional box ID, which you can use to match this movement to any messages received from the Push Pull Notifications API.
```

Comment:
Suggestion:
A successful response will contain a URI that you can use to retrieve the message status.

The issue here is grammar around usage of that/which. "That" is used in a defining sense whereas "which" is usually used in a commenting sense. If "which" is a better choice in a sentence, it must usually be prefixed by a comma. In this case, I think that "that" is a better fit for the sentence.